source/retryable-reads/tests/README.rst

```diff
@@ -25,8 +25,8 @@ versions 4.0 or later.
 
 N.B. The spec specifies 3.6 as the minimum server version: however,
 ``failCommand`` is not supported on 3.6, so for now, testing requires MongoDB
-4.0. Once `DRIVERS-560`_ is resolved, tests will be added to allow testing
-Retryable Reads on MongoDB 3.6.
+4.0. Once `DRIVERS-560`_ is resolved, live failure integration tests will be
+added to test Retryable Reads on MongoDB 3.6.
```

Comment:
This was just cut from scope of [DRIVERS-560](https://jira.mongodb.org/browse/DRIVERS-560). The initial implementation of that spec will likely end up spamming a cluster with writes and ensuring that the sequence can survive one or more failovers.

If you need 3.6 testing for retryable reads, that may need to be added as a prose test.

Reply:
I would prefer not to add a prose test for 3.6 in this review. Instead I think we should publish the spec without testing 3.6 and implement 3.6 testing as follow on work in https://jira.mongodb.org/browse/SPEC-1218.

Reply:
Ah okay, so no reads will be tested during DRIVERS-560, acknowledged. That being the case, I would also prefer to implement the 3.6 test as part of SPEC-1218,  adapting the tests that were developed as part of DRIVERS-560,